README.md

```diff
@@ -323,52 +323,53 @@ nx build dialtone-documentation
 
 ### Releasing
 
-Running these commands will call [release.sh](./scripts/release.sh) which
-automatically release all packages that need to be released.
+Currently, Dialtone packages are being release in two different ways: `scheduled` and `manually`.
+The `scheduled` release will only release changes to `production` while `manually` you can choose to release
+`alpha`, `beta` or `next` branches.
 
-Once done, a GitHub Action will be triggered, you can check the progress here:
-[release.yml](https://github.com/dialpad/dialtone/actions/workflows/release.yml)
+#### Production
 
-If something goes wrong, and you need to re-run the release, you can run the workflow manually through the `Run workflow`
-option on GitHub.
+##### Scheduled
 
-Select the `production` branch and pass the `commit SHA` which you would like to be the base to detect the changes or
-choose a `package` to release individually.
+On every Tuesday at 10:00 am UTC, [release action](.github/workflows/release.yml) will trigger the production release process which
+automatically release all packages that need to be released following the next steps:
```

Comment:
How we can stop or deactivate this automatic release?

Reply:
It's possible to disable the workflow manually, we can do that if we don't want to release that week.

![Screenshot 2024-05-14 at 9 50 16 a m](https://github.com/dialpad/dialtone/assets/87546543/37d4a55a-616b-4757-9bfc-0c5c3ea02419)